cue/schemas/panels/status-history/migrate.cue

```diff
@@ -0,0 +1,10 @@
+ if #panel.type != _|_ if #panel.type == "status-history" {
+	kind: "StatusHistoryChart"
+	spec: {
+		// simple legend
+		#showLegend: *#panel.options.legend | true
+		if #panel.options.legend != _|_ if #showLegend {
+			legend: {}
+		}
```

Comment:
I know this isnt specific to the migration logic you added there but imho this would be clearer like this:
```suggestion
		#showLegend: *#panel.options.legend | false
		if #showLegend {
			legend: {}
		}
```